installers/docker-compose/legend-with-remote-gitlab/README.md

```diff
@@ -1,4 +1,4 @@
-# Docker Compose for Legend with remote instance of Gitlab
+# Docker Compose for full Legend stack with remote instance of Gitlab
```

Comment:
Yeah. The READMEs are confusing. I had to read it several times. So we have 

1/ docker-compose/legend - This runs all the components as containers 
2/ docker-compose/legend-with-remote-gitlab - This runs all the components as containers except Gitlab
3/ docker-compose/legend-studio-dev - This runs all the components as containers except Gitlab and Studio 

Maybe instead of describing how these are implemented we should describe what they are meant for ?
Something like 

1 / docker-compose/legend - Demonstrates running Legend components as Docker containers 
2/ docker-compose/legend-with-remote-gitlab - Demonstrates the use of Gitlab.com 
3/ docker-compose/legend-studio-dev - Demonstrates using a local Legend installation to help with Studio development ?

Reply:
Also, the top level docker-compose/README can be made more clear.